src/plugins/passwd/README.md

```diff
@@ -0,0 +1,41 @@
+- infos = Information about the passwd plugin is in keys below
+- infos/author = Name <name@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = recommended productive maintained reviewed conformant compatible coverage specific unittest shelltest tested nodep libc configurable final preview memleak experimental difficult unfinished old nodoc concept orphan obsolete discouraged -1000000
+- infos/metadata =
+- infos/description =
+
+## Introduction ##
+
+This plugin parses `passwd` files, e.g. `/etc/passwd`. If present, the not-posix complient `fgetpwent` function will be used to read the file supplied by the resolver, if not, `getpwent` will be used. For writing, if present, `putpwent` will be used, if not a simple implementation writing straight to the config file.
```

Comment:
From this I would say it should always work (and no ifdef in the unit test should be needed).

Everything but the first sentence is implementation detail, not Introduction.